test/models/asyncapi_test.js

```diff
@@ -73,6 +73,17 @@ describe('AsyncAPIDocument', () => {
     });
   });
 
+  describe('#hasVariable()', function () {
+    it('should return a boolean indicating if the AsyncAPI document has server variables', () => {
+      const doc = { servers: { test1: { url: 'test1', variables: { port: { desc: 'test1' } } } } };
```

Comment:
```suggestion
      const doc = { servers: { test1: { url: 'test1:{port}', variables: { port: { desc: 'test1' } } } } };
```

I know this is not strictly needed but would be good for clarity.